diff --git a/dist/dexie-react-hooks.mjs b/dist/dexie-react-hooks.mjs
index a9e204f0f05ec9c7d9f7a598e9158efb3d38e0dc..008af4e4563f9431418eaabcb93bc0b26bb52688 100644
--- a/dist/dexie-react-hooks.mjs
+++ b/dist/dexie-react-hooks.mjs
@@ -1,4 +1,4 @@
-import { Dexie } from 'dexie';
+import dexie from 'dexie';
 import React from 'react';
 
 function useObservable(observableFactory, arg2, arg3) {
@@ -98,7 +98,7 @@ function useObservable(observableFactory, arg2, arg3) {
 }
 
 function useLiveQuery(querier, deps, defaultResult) {
-    return useObservable(function () { return Dexie.liveQuery(querier); }, deps || [], defaultResult);
+    return useObservable(function () { return dexie.Dexie.liveQuery(querier); }, deps || [], defaultResult);
 }
 
 function usePermissions(firstArg, table, obj) {
@@ -118,7 +118,7 @@ function usePermissions(firstArg, table, obj) {
         db = firstArg;
     }
     else {
-        if (firstArg instanceof Dexie)
+        if (firstArg instanceof dexie.Dexie)
             throw new TypeError("Invalid arguments to usePermission(db, table, obj): Missing table and obj arguments.");
         if (typeof firstArg.table === 'function' &&
             typeof firstArg.db === 'object') {
